src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/scaled_dot_product_attention.cpp

```diff
@@ -118,6 +124,11 @@ void ScaledAttnLayerGPUTest::SetUp() {
     for (size_t i = 0; i < inputParams.size(); i++) {
         inputParams_transpose.push_back(inputParams[i]);
     }
+    if (has_scale && is_scale_const) {
+        auto scale_const = std::make_shared<ov::op::v0::Constant>(inType, ov::Shape({1}), 0.35f);
+        scale_const->set_friendly_name("scale");
+        inputParams_transpose.push_back(scale_const);
+    }
```

Comment:
nit: could you cleanup this condition? This seems to be related to the conditions above, but why is it placed in a separate place? In addition, do we need to have three distinct case of scale handling?

Reply:
As discussed offline, I've organized the code at least for the part that handles param vector.